### PR TITLE
Chore/rls ai assistant updates 041223

### PR DESCRIPTION
**Pull request by joshenlim**
- Send policy definition if editing a policy to ai assistant when editing
- Only send entity definitions if opted into ai sql

### CODE REVIEW

File: apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/AIPolicyEditorPanel.utils.ts [hunk 1]
```diff
@@ -74,3 +74,13 @@ COMMIT;
 `.trim()
   }
 }
+
+export const generatePolicyDefinition = (policy: PostgresPolicy) => {
+  return `
+CREATE POLICY "${policy.name}" on "${policy.schema}"."${policy.table}"
+AS ${policy.action} FOR ${policy.command}
+TO ${policy.roles.join(', ')}
+USING (${policy.definition})${policy.check === null ? ';' : ''}
```
Comment (alaister) [inline apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/AIPolicyEditorPanel.utils.ts:83]: Do policies always have a using definition? For example `INSERT` expressions only have `WITH CHECK` and no `USING`

Comment (joshenlim) [reply apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/AIPolicyEditorPanel.utils.ts:83]: ahhh good point - lemme update

**Comment by joshenlim** [comment]
Also added the collapsible policy details in here - pardon the extra hacky height rendering, i can't find a clean solution for this at the moment unless anyone might have some ideas

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**